kaos/models.py

```diff
@@ -0,0 +1,52 @@
+""""Describes the object representation of tables in the database."""
+
+from flask_sqlalchemy import SQLAlchemy
+
+DB = SQLAlchemy()
+
+class ResponseHistory(DB.Model):
+    """This table serves as a generic cache to save the results of past requests so that they can be
+    quickly retrieved in the future.
+    """
+    __tablename__ = 'ResponseHistory'
+
+    uid = DB.Column(DB.Integer, primary_key=True)
+    response = DB.Column(DB.String, nullable=False)
+
+    def __init__(self, response):
+        self.response = response
+
+    def __repr__(self):
+        return '<Response History {}>'.format(self.uid)
+
+class SatelliteInfo(DB.Model):
+    """This table holds all information about a specific satellite.
+
+    The table holds the following information:
+        uid:                Unique ID for the satellite
+        name:               The name of the satellite
+        timeposvel_records: Satellite ephemeris records
+    """
+    __tablename__ = 'SatelliteInfo'
+
+    uid = DB.Column(DB.Integer, primary_key=True)
+    name = DB.Column(DB.String(50), nullable=False)
+    timeposvel_records = DB.relationship("TimePosVel", backref='satellite_info', lazy=True)
+
+class SatelliteTimPosVel(DB.Model):
```

Comment:
This name is a bit of a mouthful. Can we call it something simpler like "SatelliteOrbit" or "OrbitInfo" or even just "Orbit" instead?

Reply:
Sure